Oracle/pcornet_loader.sql

```diff
@@ -82,11 +82,11 @@ begin
   where ndc in ('00000000000', '99999999999') or length(ndc)<11 or ndc like '00NDL%' or ndc like '00SYR%'
   ;
 
- alter table pcornet_cdm.encounter
- modify facility_location varchar2(5);
+ EXECUTE IMMEDIATE 'alter table pcornet_cdm.encounter
+ modify facility_location varchar2(5)';
```

Comment:
It seems more straightforward to fix the `facility_location` definition in `PCORNetInit.sql`.